src/backend/src/services/change-requests.services.ts

```diff
@@ -429,6 +438,11 @@ export default class ChangeRequestsService {
       throwIfUncheckedDescriptionBullets(wbsElement.workPackage);
     }
 
+    const { changeRequests } = wbsElement;
+    if (!allChangeRequestsReviewed(changeRequests)) {
```

Comment:
we should only check this for change requests that haven't been deleted yet (in both places). If a CR is deleted, it's basically in a resolved state